mypy/messages.py

```diff
@@ -27,6 +27,7 @@
 MISSING_RETURN_STATEMENT = 'Missing return statement'
 INVALID_IMPLICIT_RETURN = 'Implicit return in function which does not return'
 INCOMPATIBLE_RETURN_VALUE_TYPE = 'Incompatible return value type'
+RETURN_ANY = 'Returning Any from function declared with return type {}'
```

Comment:
Normally, types and type names are put in quotes, e.g. like this: ``'... with return type "{}"'``